modules/kuberay-cluster/main.tf

```diff
@@ -44,6 +44,7 @@ resource "helm_release" "ray-cluster" {
       security_context                  = local.security_context
       secret_name                       = var.db_secret_name
       cloudsql_instance_connection_name = local.cloudsql_instance_connection_name
+      image                             = var.use_custome_image ? "us-central1-docker.pkg.dev/ai-on-gke/rag-on-gke/ray-image" : "rayproject/ray"
       image_tag                         = var.enable_gpu ? "2.9.3-py310-gpu" : "2.9.3-py310"
```

Comment:
Are we publishing both 2.9.3-py310-gpu and 2.9.3-py310?

Reply:
Both are available just in case, but I think we should limit it to just the 2.9.3-py310-gpu. I'll make the change to ensure that if the use_custom_image is enabled, it will always use the 2.9.3-py310-gpu